### PR TITLE
README for cAdvisor configs

### PR DESCRIPTION
**Pull request by wacuuu**

### CODE REVIEW

File: examples/kubernetes/monitoring/cadvisor/README.rst [hunk 1]
```diff
@@ -0,0 +1,47 @@
+=====================================================
+Migration to cAdvisor
+=====================================================
+
+**This part is curently developed and is missing proper testing and validation**
```
Comment (Creatone) [inline examples/kubernetes/monitoring/cadvisor/README.rst:5]: Typo `curently` -> `currently`

File: examples/kubernetes/monitoring/cadvisor/README.rst [hunk 2]
```diff
@@ -0,0 +1,79 @@
+=====================================================
+Migration to cAdvisor
+=====================================================
+
+**This part is currently developed and is missing proper testing and validation**
+
+Kubernetes deployment
+=====================
+
+There is a ready to use example of cAdvisor deployment with perf support available `on cAdvisor repo as overlay example <https://github.com/google/cadvisor/tree/master/deploy/kubernetes/overlays/examples_perf>`_. It is advised to change the cpu limitations in daemonset definition, as prometheus client, which is serving the stats, tends to generate a lot of CPU load with growth of stats size.
```
Comment (katarzyna-z) [inline examples/kubernetes/monitoring/cadvisor/README.rst:10]: In my opinion it is better to indicate [this](https://github.com/google/cadvisor/tree/master/deploy/kubernetes#cadvisor-with-perf-support-on-kubernetes) section in documentation than directory with set of yaml files. User may not know what to do with them.

File: examples/kubernetes/monitoring/cadvisor/README.rst [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+- `id` it identifies the continer by mix of pod id and docker container id
+- `name` it identifies the container by container name asigned by k8s
+
+
+Building cAdvisor with perf
+===========================
+
+As specified in `official build instruction <https://github.com/google/cadvisor/blob/master/docs/development/build.md#perf-support>`_, assuming that ones environment is capable of building go packages, the following steps have to be done:
+
+.. code-block:: shell
```
Comment (katarzyna-z) [inline examples/kubernetes/monitoring/cadvisor/README.rst:55]: This section is not visible in browser.

File: examples/kubernetes/monitoring/cadvisor/README.rst [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+  apt-get install libpfm4 libpfm4-dev
+  git clone https://github.com/google/cadvisor.git
+  cd cadvisor
+  GO_FLAGS="-tags=libpfm,netgo" make build
+  # To test if the binary has compiled properly
+  sudo ./cadvisor
+
+Then, to check if cadvisor is responding, from other terminal on the same machine execute
+
+.. code-block:: shell
```
Comment (katarzyna-z) [inline examples/kubernetes/monitoring/cadvisor/README.rst:66]: This section is also not visible in browser.

File: examples/kubernetes/monitoring/cadvisor/README.rst [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+  curl localhost:8080/metrics
+
+If there is a response it means, that cadvisor was succesfuly compiled and is running properly
+
+cAdvisor in docker container
+============================
+
+Go to cAdvisor repo root directory and run
+
+.. code-block:: shell
```
Comment (katarzyna-z) [inline examples/kubernetes/monitoring/cadvisor/README.rst:76]: Not visible in browser

File: examples/kubernetes/monitoring/cadvisor/README.rst [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+
+
+
+Two types can be distinguished:
```
Comment (katarzyna-z) [inline examples/kubernetes/monitoring/cadvisor/README.rst:32]: FYI perf uncore metrics are now also available in cAdvisor, description for them will be added soon, see: https://github.com/katarzyna-z/cadvisor/pull/17

File: examples/kubernetes/monitoring/cadvisor/README.rst [hunk 7]
```diff
@@ -0,0 +1,124 @@
...
+  --volume=$PWD/perf-prm-skylake.json:/etc/configs/perf/perf-prm-skylake.json \
+  --publish=8080:8080 \
+  --device=/dev/kmsg \
+  --privileged \
+  --name=cadvisor \
+  cadvisor --perf_events_config=/etc/configs/perf/perf-prm-skylake.json
+
+Important note is that it should be run on Skylake platform, as some of the metrics in mentioned json are only available on Skylake. After this, command:
+
+.. code-block:: shell
```
Comment (katarzyna-z) [inline examples/kubernetes/monitoring/cadvisor/README.rst:109]: Not visible in browser

**Pull request merged**